packages/text-input-react/example/index.tsx

```diff
@@ -23,6 +23,7 @@ const TextFieldDemo = () => {
                     placeholder={"Norsk"}
```

Comment:
I'm not sure if it is a good idea, but since we use the spread operator `{...rest}` we can remove many fields:

```
    return (
        <label data-testid="jkl-text-field" className={componentClassName}>
            <span className={labelClassName}>{label}</span>
            <input aria-invalid={!!errorLabel} className={`jkl-text-field__input`} {...rest} />
            <SupportLabel helpLabel={helpLabel} errorLabel={errorLabel} />
        </label>
    );

```

Just throwing it out there.
```